x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -156,9 +166,68 @@ export async function installKibanaAssetsAndReferences({
     kibanaAssets,
   });
 
+  await tagKibanaAssets({
+    savedObjectTagAssignmentService,
+    savedObjectTagClient,
+    kibanaAssets,
+    pkgTitle,
+  });
+
   return installedKibanaAssetsRefs;
 }
 
+const TAG_COLOR = '#FFFFFF';
+const MANAGED_TAG_NAME = 'Managed';
+
+export async function tagKibanaAssets({
```

Comment:
Do you think it'd be possible to wrap this in an APM span and track how it impacts installation performance for packages?